inc/telemetry/namespace.php

```diff
@@ -299,12 +299,17 @@ function get_environment_details() : array {
  * @return void
  */
 function load_segment_js() {
+	if ( isset( $_GET['page'] ) ){
+		$page_name = str_replace( '-', ' ', ucfirst( $_GET['page'] ) );
```

Comment:
I think there has to be a way to get the page title passed through in the `add_menu_page()` / `add_menu_subpage(0` functions.. can you look into that? Maybe using `get_current_screen()` and the `WP_Screen` object?

I'm never keen on approaches like this using functions like `ucfirst` and `ucwords`, they're almost never appropriate if there's a genuine human readable string available somewhere.

Reply:
I haven't found an alternative way to get the the page title with your suggestions.

What if we stuck with using `get_admin_page_title();` for the title, then use the the filter `admin_title` filter to change the title as and when needed?

Each of the pages in the documents when converting the markdown and an `h1` tag is set, it then overrides the original page title.

Reply:
> What if we stuck with using get_admin_page_title(); for the title, then use the the filter admin_title filter to change the title as and when needed?

Yep agreed, that's the approach I'm after here. The only special case we have is our replacement dashboard - I'll make a PR to filter that and tag you for review. Then the implementation here can just be to use `get_admin_page_title()`.

Noticed a general issue with the admin docs page titles that it'd be good to fix to make this more useful https://github.com/humanmade/altis-documentation/issues/397

Reply:
So that filter and the function are a little strange - the filter modifies the entire title string, including the site name, while the function sets the default value for the `$title` global variable, unless one has already been set.